src/vue/common/EmailGetter.vue

```diff
@@ -6,6 +6,12 @@
   >
     {{ result }}
   </span>
+  <span
+    v-else-if="isLoadingFailed"
+    class="email-getter"
+  >
+    {{ FALLBACK_MESSAGES.NO_ACCOUNT_OR_BALANCE_ID }}
```

Comment:
you  do not need `FALLBACK_MESSAGES` anymore. You can use raw m-dash (`&mdash;`) here.

Reply:
Resolved, updated